cmd/stack.go

```diff
@@ -82,7 +85,7 @@ func setupStackCommand() *cobra.Command {
 	}
 	upCommand.Flags().BoolP(cobraext.DaemonModeFlagName, "d", false, cobraext.DaemonModeFlagDescription)
 	upCommand.Flags().StringSliceP(cobraext.StackServicesFlagName, "s", nil,
-		fmt.Sprintf(cobraext.StackServicesFlagDescription, strings.Join(availableServicesAsList(), ", ")))
+		fmt.Sprintf(cobraext.StackServicesFlagDescription, `"`+strings.Join(availableServicesAsList(), ",")+`"`))
```

Comment:
Could you change the format defined in `StackServicesFlagDescription` instead of concatenating `"`?